src/main/resources/com/devonfw/ide/sonarqube/common/rules/devon4j/E7_java.html

```diff
@@ -0,0 +1,52 @@
+<p>
+    This rule checks that all Use-Case implementation methods are annotated with a security constraint.
+    Valid security constraint annotations are the following from the javax.annotation.security package:
+    <ul type='disc'>
+        <li>@DenyAll</li>
+        <li>@PermitAll</li>
+        <li>@RolesAllowed</li>
+    </ul>
+</p>
+<h2>Noncompliant Code Example</h2>
+<pre>
+    public class TestInterfaceImpl implements TestInterface {
+
+        @Override                                 
+        private boolean testBoolMethod() {
```

Comment:
Same as with test examples:
a private method can not override. Change to public to give a consistent example.